volunteer-communication-center/assets/studio_flow.private.js

```diff
@@ -0,0 +1,397 @@
+const flowDefinition = {
+  "description": "VoiceToFlex",
+  "states": [
+    {
+      "name": "Trigger",
+      "type": "trigger",
+      "transitions": [
+        {
+          "next": "autopilot_2",
+          "event": "incomingMessage"
+        },
+        {
+          "next": "autopilot_1",
+          "event": "incomingCall"
+        },
+        {
+          "event": "incomingRequest"
+        }
+      ],
+      "properties": {
+        "offset": {
+          "x": 10,
+          "y": 10
+        }
+      }
+    },
+    {
+      "name": "autopilot_1",
+      "type": "send-to-auto-pilot",
+      "transitions": [
+        {
+          "next": "split_1",
+          "event": "sessionEnded"
+        },
+        {
+          "event": "failure"
+        },
+        {
+          "event": "timeout"
+        }
+      ],
+      "properties": {
+        "chat_channel": "{{trigger.message.ChannelSid}}",
+        "offset": {
+          "x": 170,
+          "y": 210
+        },
+        "autopilot_assistant_sid": "UA232372d4ebf310e2eedd7bcc099966e1",
```

Comment:
There are a number of these hard coded `SID` values in this file that are unique to the account which created them. As a more generic template for others to use, you will need to ensure the custom info per account is used instead.

I would advise you move this file from the `assets` folder and make it a `function` file so that it can access the `context` which hopefully has the appropriate values from the .env file you need to populate those values. You can then continue to have it insert those values dynamically as part of its response as a function.

The other impacted lines are 104, 105, 180, 308, 358, 359, and 383